sphinx/config.py

```diff
@@ -254,6 +254,7 @@ class Config:
         'smartquotes_excludes': _Opt(
             {'languages': ['ja'], 'builders': ['man', 'text']}, 'env', ()),
         'option_emphasise_placeholders': _Opt(False, 'env', ()),
+        'enable_parallel_post_transform': _Opt(False, 'html', ()),
```

Comment:
Should we use 'html' or 'env' ? (I always forget how it works).

Reply:
Currently yeh it should be `env`, ideally it would a value to signify that all output formats should be rebuild (whilst still using the cached doctrees and env) but this does not currently exist